src/components/tool/guide/EresearchJob/EresearchJob.js

```diff
@@ -100,6 +100,34 @@ const getConfigGroups = (config, onConfigChange = () => {}) => {
         selected: (config) => isValidChoice(jobTypes, config?.jobType),
       };
     },
+    resources: () => {
+      return {
+        element: (key, selected) => (
+          <ConfigPicker
+            key={key}
+            title="Resources"
+            description="The resources available on the nodes used by the job."
+            selected={selected}
+            inputProps={{
+              choices: [
+                ["Automatic", "Run the job without specifying resources"],
+                ["Custom", "Specify the resources to use for the job"],
```

Comment:
Just a thought: Is `to use` necessary? I think `Specify the resources for the job (manually)` would be sufficient?

Reply:
Clarified the wording